src/lib/registration/connect_helpers.rb

```diff
@@ -266,9 +264,9 @@ def self.import_ssl_certificate(cert)
         # imported but the registration server still cannot be accessed securely,
         # user has to solve the certificate issue manually.
         Yast::Report.Error(_("A certificate has been already imported\n" \
-          "but the server connection still cannot be trusted.\n\n" \
-          "Please fix the certificate issue manually, ensure that the server\n" \
-          "can be connected securely and start the YaST module again."))
+                             "but the server connection still cannot be trusted.\n\n" \
```

Comment:
I'm not sure if I like this extra indentation for texts.  It makes less space available and requires more shorter lines. :thinking: 

Esp. it is strange that for Hashes or named parameters it requires the opposite.